app/Console/UpdateOfferStatusCommand.php

```diff
@@ -61,32 +62,48 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $status = new Status($statusType, $reasons);
 
+        if ($count <= 0) {
+            $output->writeln('Query found 0 ' . $this->getPluralOfferType() . ' to update');
+            return 0;
+        }
+
         $confirmation = $this->getHelper('question')
             ->ask(
                 $input,
                 $output,
                 new ConfirmationQuestion(
                     "This action will update the status of {$count} {$this->getPluralOfferType()} to {$statusType->toNative()}, continue? [y/N] ",
-                    true
+                    false
                 )
             );
 
         if (!$confirmation) {
             return 0;
         }
 
+        $exceptions = [];
         $offers = $this->searchResultsGenerator->search($query);
         $progressBar = new ProgressBar($output, $count);
 
         foreach ($offers as $id => $offer) {
-            $this->commandBus->dispatch(
-                new UpdateStatus($id, $status)
-            );
+            try {
+                $this->commandBus->dispatch(
+                    new UpdateStatus($id, $status)
+                );
+            } catch (Exception $exception) {
+                $exceptions[$id] = 'Offer with id: ' . $id . ' caused an exception: ' . $exception->getMessage();
+            }
+
             $progressBar->advance();
         }
 
         $progressBar->finish();
 
+        $output->writeln('');
+        foreach ($exceptions as $exception) {
+            $output->writeln($exception);
+        }
+
```

Comment:
Just thinking out loud here: We do the reporting of these exceptions in loops differently in the various CLI commands that do bulk operations. Maybe we should look for a more consistent approach sometime? Not in scope of this PR of course, but something to think about maybe.

Reply:
Yeah, I first wanted to make it consistent.

But I think it can be different based on the CLI command:
- Is it something that processes a lot of data?
- Is execution manual or automated?
- Do we gather only errors or all information?
- Do we want to save the gathered feedback?

So a good approach could be to create a `Feedback` class that gathers the information and/or errors and has a few output methods to stream to file our console output.